opentelemetry-zipkin/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## vNext
 
+### Changed
+
+- Update examples to `hyper` v1 [#1674](https://github.com/open-telemetry/opentelemetry-rust/pull/1674)
```

Comment:
nit - this doesn't look to be user impacting update, can be removed from changelog ?

Reply:
Referring to the examples here is a little weird, but opentelemetry-zipkin does expose the `HyperClient` trait in its `ZipkinPipelineBuilder` API so IMO there should be some mention of this change.

Reply:
Does it though? I haven't found the point where it's exposed that you're referring to

Reply:
Oh, do you mean `HttpClient`? Fair. Yeah.

Reply:
Sorry, yes, `HttpClient`.

Reply:
Adjusted the changelog entry. Let me know what you think.

Reply:
LGTM!